pkg/controller/networkpolicy/validate.go

```diff
@@ -213,6 +214,36 @@ func (v *NetworkPolicyValidator) validateAntreaPolicy(curObj, oldObj interface{}
 	return reason, allowed
 }
 
+// validatePort validates if ports is valid
+func (a *antreaPolicyValidator) validatePort(ingress, egress []secv1alpha1.Rule) error {
+	isValid := func(rules []secv1alpha1.Rule) error {
+		for _, rule := range rules {
+			for _, port := range rule.Ports {
+				if port.EndPort == nil {
+					continue
+				}
+				if port.EndPort != nil && port.Port == nil {
```

Comment:
the reason why we added L222 is that you no longer need to check for port.EndPort != nil since if it was, you'd not reach this point :)
here and below

Reply:
Ah. Got you.